src/dcd/server.ts

```diff
@@ -46,7 +48,7 @@ export default class Server extends vsc.Disposable {
             additionsImports.push('-I' + item);
         })
 
-        if (process.platform === 'windows') {
+        if (process.platform === 'win32') {
             additionsImports.push('-IC:\\D\\dmd2\\src\\phobos');
```

Comment:
@mattiascibien I add somewhat forgotten about this, maybe adding this to the extension configuration would be better